web-api/src/ingestion/routes.rs

```diff
@@ -46,34 +43,31 @@ pub(super) fn configure_service(config: &mut ServiceConfig) {
     config
```

Comment:
:art: since we already touch the code here, we could make it a bit less verbose, instead of `config.service(resource::route())` we could use the `config.route()` alias, like
```rust
config
    .route("/candidates", web::get().to(get_document_candidates))
    .route("/candidates", web::put().to(set_document_candidates))
    // etc
```
similarly for the personalization routes.

Reply:
I'm not sure that is simpler as it duplicates all the paths. If we do that we could also use the `#[get(...)]` etc. macros.